### ISSUE TITLE
Does not work correctly on Firefox Send

### ISSUE THREAD

**Issue by MikkCZ** [mozilla-services/screenshots#3256]
STR:
1) Open send.firefox.com.
2) Click the icon in toolbar to take a screenshot.
3) Select any area, either by explicit mouse select or just by clicking and element.
4) Click the download button.
5) The saved screenshot contains a different page area.

Mozilla/5.0 (X11; Linux x86_64; rv:56.0) Gecko/20100101 Firefox/56.0 ID:20170801100311 CSet: 51ffb9283f0c7c00e08eb8c39b33fbee218c370d

**Comment by wresuolc** [comment]
Can you be more specific about what it's saving?  (Can you link to your screenshot?)

Do you have any other add-ons installed?

We can't reproduce this so far.

**Comment by MikkCZ** [comment]
I am able to reproduce even with a clean profile. Here is how it looks like.

Before I select the element by clicking it:
![snimek obrazovky porizeny 2017-08-01 17-45-14](https://user-images.githubusercontent.com/3941344/28834098-a6323c22-76e1-11e7-8178-e95c9df9afc3.png)
After I select the element by clicking it:
![snimek obrazovky porizeny 2017-08-01 17-45-21](https://user-images.githubusercontent.com/3941344/28834208-eff9cc9e-76e1-11e7-96c4-a159c50894f9.png)

What I manually select:
![snimek obrazovky porizeny 2017-08-01 17-45-49](https://user-images.githubusercontent.com/3941344/28834272-14972236-76e2-11e7-9283-452a660e1cf9.png)
What is saved than:
![screenshot-2017-8-1 firefox send](https://user-images.githubusercontent.com/3941344/28834293-23760ace-76e2-11e7-83ba-4775d7304d62.png)

**Comment by MikkCZ** [comment]
OK, the first scenario by selecting with mouse click on the element works and the saved image looks as expected. The visualization of the selected area is only broken (second screenshot above).

The saved image is only broken for the manual area selection (screenshots three and four), where it captures something different.

**Comment by johngruen** [comment]
@SoftVision-CosminMuntean could you help us verify?

**Comment by SoftVision-CosminMuntean** [comment]
I have managed to reproduce this issue on latest Nightly build. It seems that when you made a selection, the Screenshots overlay is pushed to the right, and also the position of the selected area is moved.

I have managed to reproduce this also on Mac OS, but it seems that on smaller screens (eg: 13 inches) the issue is not reproducible. Also, the browser has to be maximized in order to reproduce the issue.

Here is a screen recording with the issue:
![screenshots on send](https://user-images.githubusercontent.com/19226534/28972378-21009e14-7938-11e7-9f36-cdb1f98df385.gif)
